src/components/Tooltip/Tooltip.js

```diff
@@ -418,6 +418,7 @@ export default class Tooltip extends Component {
               onBlur={evt => this.handleMouse(evt)}
               aria-haspopup="true"
               aria-expanded={open}
+              style={this.props.style}
```

Comment:
Can we do (probably additional) CSS classes rather than `style` attribute?

Reply:
the PR is open here: https://github.com/IBM/carbon-components/pull/1635